src/README.md

```
@@ -159,3 +159,127 @@ components:
 
 [<img src="https://cloudposse.com/logo-300x69.svg" height="32" align="right"/>](https://cpco.io/homepage?utm_source=github&utm_medium=readme&utm_campaign=cloudposse-terraform-components/aws-alb&utm_content=)
 
+
+<!-- BEGIN_TF_DOCS -->
+## Requirements
```

markdownlint-cli2 (0.23.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)